### PR TITLE
Landing-page cleanup

### PR DESCRIPTION
**Pull request by epwinchell**

### CODE REVIEW

File: src/components/app-content-renderer/second-panel.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
 } from '@patternfly/react-core';
 import React from 'react';
 
-import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+import { ArrowRightIcon } from '@patternfly/react-icons';
```
Comment (Hyperkid123) [inline src/components/app-content-renderer/second-panel.js:14]: Can you please keep the absolute path? Tree shaking does not work on the Icons package and it means we will pull all 2k icons in the production bundle.

Comment (Hyperkid123) [reply src/components/app-content-renderer/second-panel.js]: I see. can you switch the `/esm/` from `/js/` in the import path? It will fix the problem.

Comment (epwinchell) [reply src/components/app-content-renderer/second-panel.js]: @Hyperkid123 There's a new warning, but Travis is green.

```
    Warning: An update to FirstPanel inside a test was not wrapped in act(...).
      
      When testing, code that causes React state updates should be wrapped into act(...):
      
      act(() => {
        /* fire events that update state */
      });
      /* assert on the output */
      
      This ensures that you're testing the behavior the user would see in the browser. Learn more at https://reactjs.org/link/wrap-tests-with-act
          at FirstPanel (/Users/epwinchell/work/landing-page-frontend2/src/components/app-content-renderer/first-panel.js:8:27)
          at div
          at SplitItem (/Users/epwinchell/work/landing-page-frontend2/node_modules/@patternfly/react-core/src/layouts/Split/SplitItem.tsx:15:3)
          at div
          at Split (/Users/epwinchell/work/landing-page-frontend2/node_modules/@patternfly/react-core/src/layouts/Split/Split.tsx:19:3)
          at Landing
          at Router (/Users/epwinchell/work/landing-page-frontend2/node_modules/react-router/cjs/react-router.js:99:30)
          at MemoryRouter (/Users/epwinchell/work/landing-page-frontend2/node_modules/react-router/cjs/react-router.js:187:35)
          at Provider (/Users/epwinchell/work/landing-page-frontend2/node_modules/react-redux/lib/components/Provider.js:21:20)
          at ScalprumProvider (/Users/epwinchell/work/landing-page-frontend2/node_modules/@scalprum/react-core/src/scalprum-provider.tsx:20:9)
          at LandingWrapper (/Users/epwinchell/work/landing-page-frontend2/src/routes/__tests__/Landing.test.js:12:27)
```

Comment (Hyperkid123) [reply src/components/app-content-renderer/second-panel.js]: Yeah that warning is "fine" but yeah things that change state should be wrapped in the act function: https://reactjs.org/docs/test-utils.html#act

Its not that big of a deal

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**